subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/ArchivesContinuousIntegrationTest.groovy

```diff
@@ -123,6 +125,7 @@ class ArchivesContinuousIntegrationTest extends Java7RequiringContinuousIntegrat
     }
 
     @Ignore("inputs from resources are ignored")
+    @Unroll
```

Comment:
No, it's fine.  I thought maybe it was hinting at something blowing up with the new annotation if it didn't use `@Unroll`.